packages/@atjson/renderer-plain-text/test/text-renderer-test.ts

```diff
@@ -35,4 +37,21 @@ describe("PlainTextRenderer", () => {
     let text = PlainTextRenderer.render(doc);
     expect(text).toBe("This is some fancy text.");
   });
+
+  it("renders line breaks", () => {
+    let document = new OffsetSource({
+      content: "first linesecond line",
```

Comment:
```suggestion
      content: "first line\uFFFCsecond line",
```

Reply:
hmm doesn't seem to work either, it inserts a space i assume? 

<img width="251" alt="Screen Shot 2020-09-23 at 2 11 58 PM" src="https://user-images.githubusercontent.com/15381728/94052746-5d020280-fda7-11ea-94b8-973f58328276.png">